packages/yoastseo/src/worker/AnalysisWebWorker.js

```diff
@@ -1089,7 +1090,7 @@ export default class AnalysisWebWorker {
 			this._researcher.setPaper( this._paper );
 
 			const languageProcessor = new LanguageProcessor( this._researcher );
-			this._paper.setTree( build( this._paper.getText(), languageProcessor ) );
+			this._paper.setTree( filterTree( build( this._paper.getText(), languageProcessor ), permanentFilters ) );
```

Comment:
I am a bit in doubt where to actually apply the filter. But in AnalysisWebworker seemed the best location for me.